routes/user.routes.js

```diff
@@ -73,4 +82,43 @@ router.post("/:folderId/edit", (req, res, next) => {
     .catch((err) => next(err));
 });
 
+router.post("/newtab", async (req, res, next) => {
+  const { category, description } = req.body;
+
+  const link = await axios.post("https://api.linkpreview.net", {
+    q: `${description}`,
+    key: `${process.env.LINK_KEY}`,
```

Comment:
No need to use string interpellation as this is already a javascript variable. Only use it when you are combining it with text